tasks/configure.yml

```diff
@@ -12,8 +12,6 @@
      - { src: realip-snippet.conf.j2, dest: /etc/nginx/snippets/realip.conf }
      - { src: fpm-snippet.conf.j2, dest: /etc/nginx/snippets/fastcgi-php.conf }
      - { src: x-whom.conf.j2, dest: /etc/nginx/snippets/x-whom.conf }
-  tags:
-    - nginx
```

Comment:
The removal of tags in this file combined with the tags in other files is pretty confusing. Should they all be removed?

Reply:
I'm just using tags in the `main.yml` file. We should only have tags on these tasks if we want to run them specifically in conjunction with another role or task file.